docs/querying/functions.md

```diff
@@ -39,6 +39,29 @@ absent(sum(nonexistent{job="myjob"}))
 In the second example, `absent()` tries to be smart about deriving labels of the
 1-element output vector from the input vector.
 
+## `absent_over_time()`
+
+`absent_over_time(v range-vector)` returns an empty vector if the range vector
+passed to it has any elements and a 1-element vector with the value 1 if the
+range vector passed to it has no elements.
+
+This is useful for alerting on when no time series exist for a given metric name
+and label combination for a certain amount of time.
+
+```
+absent_over_time(nonexistent{job="myjob"}[1h])
+# => {job="myjob"}
+
+absent_over_time(nonexistent{job="myjob",instance=~".*"}[1h])
+# => {job="myjob"}
+
+absent_over_time(sum(nonexistent{job="myjob"})[1h:])
+# => {}
+```
+
+In the second example, `absent_over_time()` tries to be smart about deriving
```

Comment:
I presume you're going to update this per your other PR?

Reply:
yes